tests/test_scripts.py

```diff
@@ -62,7 +62,24 @@ def test_eval_policy(config):
       run = eval_policy_ex.run(config_updates=config_updates,
                                named_configs=['fast'])
       assert run.status == 'COMPLETED'
-      assert isinstance(run.result, dict)
+      wrapped_reward = 'reward_type' in config
+      _check_result_rollout_stats(run.result, wrapped_reward)
+
+
+def _check_result_rollout_stats(result: dict,
+                                wrapped_reward: bool = True):
+  """Common assertions for results["rollout_stats"]."""
+  assert isinstance(result, dict)
+  stats = result.get("rollout_stats")
+  assert isinstance(stats, dict)
+  assert "return_mean" in stats
+  assert "monitor_return_mean" in stats
+  if wrapped_reward:
```

Comment:
This feels like a unit test style test (does this particular computation of monitor returns match what we expect), but it's implemented as an integration test (does the output of this *script* match what we expect). Why not perform this check on the `rollout_stats` function directly?

Reply:
The `if wrapped_reward` part seems important as a check for whether the script's venv reward function was correctly wrapped or correctly not wrapped (would be bad if this were incorrect in either direction).

This doubles as a check that `rollout_stats` works correctly under reward wrapping. If it seems helpful I could add a similar test to `tests/test_rollout.py`.

Reply:
E.G: in `test_eval_policy` this is used to verify that `reward_type` `reward_path` arguments are changing the reward.

https://github.com/HumanCompatibleAI/imitation/blob/f545fb55e6cde010003f848978e0378c6b12ce02/tests/test_scripts.py#L61-L66

Reply:
I think it would be worth adding something similar in `test_rollout`. You can chooes the wrapped reward to output a known, fixed value so you can directly test if returns are computed correctly.

Reply:
Ok, added new tests.